### PR TITLE
Make dash height work for both GridUnits and pixesl

### PR DESCRIPTION
**Pull request by Herrie82**

### CODE REVIEW

File: qml/Notifications/NotificationArea.qml [hunk 1]
```diff
@@ -93,7 +93,16 @@ Rectangle {
             var dashHeight = 0;
             if( window.windowProperties && window.windowProperties.hasOwnProperty("LuneOS_dashheight") )
             {
-                dashHeight = Units.length(window.windowProperties["LuneOS_dashheight"]);
+                //If the provide it in GridUnits we need to make sure we deal with it properly.
+                if( window.windowProperties && window.windowProperties.hasOwnProperty("LuneOS_metrics") && window.windowProperties["LuneOS_metrics"]==="units")
+                {
+                    dashHeight = Units.length(Units.gu(window.windowProperties["LuneOS_dashheight"]));
```
Comment (Herrie82) [inline qml/Notifications/NotificationArea.qml]: I'm not 100% sure if we should do Units.length here or it would then be double a bit?

Comment (Tofee) [inline qml/Notifications/NotificationArea.qml]: No, should be just Units.gu here

File: qml/Notifications/NotificationArea.qml [hunk 2]
```diff
@@ -93,7 +93,16 @@ Rectangle {
...
+                }
+                //Provided in normal pixels, convert to device independant pixels
```
Comment (Tofee) [inline qml/Notifications/NotificationArea.qml]: Actually it's the contrary, we convert it to device pixels here

File: qml/Notifications/NotificationArea.qml [hunk 3]
```diff
@@ -93,7 +93,16 @@ Rectangle {
             var dashHeight = 0;
             if( window.windowProperties && window.windowProperties.hasOwnProperty("LuneOS_dashheight") )
             {
-                dashHeight = Units.length(window.windowProperties["LuneOS_dashheight"]);
+                //If the provide it in GridUnits we need to make sure we deal with it properly.
+                if( window.windowProperties && window.windowProperties.hasOwnProperty("LuneOS_metrics") && window.windowProperties["LuneOS_metrics"]==="units")
```
Comment (Tofee) [inline qml/Notifications/NotificationArea.qml]: No need to check again for window.windowProperties, we know it's non-null. The if clause is heavy enough :)

File: qml/Notifications/NotificationArea.qml [hunk 4]
```diff
@@ -93,7 +93,16 @@ Rectangle {
             var dashHeight = 0;
             if( window.windowProperties && window.windowProperties.hasOwnProperty("LuneOS_dashheight") )
             {
-                dashHeight = Units.length(window.windowProperties["LuneOS_dashheight"]);
+                //If the provide it in GridUnits we need to make sure we deal with it properly.
+                if( window.windowProperties.hasOwnProperty("LuneOS_metrics") && window.windowProperties["LuneOS_metrics"]==="units")
+                {
+                    dashHeight = Units.length(Units.gu(window.windowProperties["LuneOS_dashheight"]));
```
Comment (Tofee) [inline qml/Notifications/NotificationArea.qml]: No Unit.length here

**Pull request merged**